### PR TITLE
ENT-32 added a helper method to retrieve EC branding details

### PR DESCRIPTION
**Pull request by asadiqbal08**
Added a helper method to retrieve EC branding details.

### CODE REVIEW

**Comment by asadiqbal08** [comment]
@saleem-latif @mattdrayer If you are ok then I am going to quick merge with this PR into Saleem's PR.

File: test_utils/factories.py [hunk 1]
```diff
@@ -127,3 +127,20 @@ class Meta(object):
...
+    """
+
+    class Meta(object):
+        """
+        Meta for EnterpriseCustomerBrandingFactory.
+        """
+
+        model = EnterpriseCustomerBrandingConfiguration
+
+enterprise_customer = factory.SubFactory(EnterpriseCustomerFactory)
```
Comment (saleem-latif) [inline test_utils/factories.py]: @asadiqbal08 I think this statement is not indented correctly, shouldn't it be part of `EnterpriseCustomerBrandingFactory ` class ?

File: tests/test_api.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    def test_enterprise_branding_info(self):
+        """
+        Test ``get_enterprise_branding_info`` helper method.
+        """
+        faker = FakerFactory.create()
+        provider_id = faker.slug()
+        customer = EnterpriseCustomerFactory()
+        EnterpriseCustomerIdentityProviderFactory(provider_id=provider_id, enterprise_customer=customer)
+        EnterpriseCustomerBrandingFactory(
+            id=1,
```
Comment (saleem-latif) [inline tests/test_api.py]: you can skip this argument if you are not planning to use it later for assertions.

File: test_utils/factories.py (hunk 1 above)
Comment (asadiqbal08) [inline test_utils/factories.py]: yup. jenkins was saying the same in quality :)

**Comment by asadiqbal08** [comment]
@mattdrayer Should I go with merge ?

**Comment by mattdrayer** [comment]
Let's have a review from @saleem-latif since his branch is the target -- I am 👍 for the change

**Pull request merged**